orchestrator/serverless.yml

```diff
@@ -111,9 +112,10 @@ functions:
               - X-Amz-Security-Token
               - X-Amz-User-Agent
               - x-requested-with
+
       - http:
-          method: POST
-          path: /order/execute
+          method: GET
+          path: /wallet/channel
```

Comment:
is this expecte change

Reply:
his is some git hub compare issue. all http event are there i just listed order events separately

Reply:
this is some git hub compare issue. all http event are there i just listed order events separately